torch/csrc/jit/passes/quantization.cpp

```diff
@@ -580,8 +629,20 @@ TORCH_API void InsertObservers(
     script::Module& module,
     const std::string& method_name,
     const QConfigDict& qconfig_dict) {
-  auto module_qconfig_map = getQConfigMap(module, qconfig_dict);
-  InsertObserversImpl(module, method_name, module_qconfig_map);
+  ModuleQConfigMap module_qconfig_map;
+  // Set of child modules that has the same qconfig
+  // as parent module, used for avoiding inserting
+  // observers for Tensors that's used in child module
+  // and produced by the child module if that module has
+  // the same qconfig as the parent module.
+  ModulePtrSet child_module_set;
+  getQConfigMap(module,
+                qconfig_dict,
+                "",
```

Comment:
These parameters should probably have default values so that this invocation looks simple. Right now the reader is unnecessarily exposed to internal details of the function implementation.

Reply:
I can make these two default